scripts/test-in-docker-locally.bash

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -eu
+
+THIS_FILE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+CI="${THIS_FILE_DIR}/../../wg-app-platform-runtime-ci"
+. "$CI/shared/helpers/git-helpers.bash"
+REPO_NAME=$(git_get_remote_name)
+
+if [[ ${DB:-empty} == "empty" ]]; then
+  DB=mysql
+fi
+
+DB="${DB}" "${THIS_FILE_DIR}/create-docker-container.bash" -d
+
+docker exec $REPO_NAME-docker-container '/repo/scripts/docker/tests-templates.bash'
+docker exec $REPO_NAME-docker-container '/repo/scripts/docker/test.bash' "$@"
```

Comment:
I'm seeing transient errors when I was testing this.

`scripts/test-in-docker-locally.bash` appears to consistently fail in the silk teardown suite (`src/code.cloudfoundry.org/silk/teardown`).

But then manual invocation of `/repo/scripts/docker/test.bash` from within a running container works just fine.